client/cypress/integration/event-manager/login_spec.js

```diff
@@ -0,0 +1,75 @@
+describe('Logging In', function() {
+
+	context('Should be on login page', function() {
+		it('successfully load login page', function() {
+      cy.visit('/login')
```

Comment:
@taleymirza @QaseemLodhi if there is no assertion, there is no point of writing that test. Also our focus should be testing the logic in a view, successful page loads are not very useful.

Reply:
Ok so should we remove this test?

Reply:
purpose of this test is to check login page is loaded.